Test.C

```diff
@@ -1,53 +1,161 @@
+#include <stdio.h>
+#include <stdlib.h>
+
 /*
 
-Hello
+defining number literals
 
 */
 
-int main(){ 
-
-    //initialize game 
+#define NAME_LENGTH 20
+#define DESCRIPTION_LENGTH 200
+#define NUM_EQUIPMENT 10
+#define NUM_ENCRYPTION 10
+#define NUM_ATTACK 5
+
+int main(){
+
+    //initialize game
+
+    struct Equipment {
+        int id;
+        char name[NAME_LENGTH]; // name of the piece of Equipment
+        char description[DESCRIPTION_LENGTH]; // short description of Equipment
+        int credits; // cost of the Equipment
+        int effectiveness; // number of messages able to be sent
+    };
+
+    struct Encryption {
+        int id;
+        char name[NAME_LENGTH]; // name of the encryption method
+        char description[DESCRIPTION_LENGTH]; // short description of how encryption works
+        int credits; // cost of the encryption
+        double effectiveness; // how effective the encryption is (percentage)
+
+    };
+
+    struct Attack {
+        int id;
+        char name[NAME_LENGTH]; // name of the attack
+        char description[DESCRIPTION_LENGTH]; // short description of how the attack works
+        int tier; // level of intensity
+
+    };
+
+    struct EquipmentInventory {
+        Equipment equipment; // the data
+        int quantity; // the number of this equipment type currently owned
+        EquipmentInventory *next; // the pointer
+    };
```

Comment:
This structure would be replaced by an array, `int equipmentInventory[]` that would keep track of each amount. The indexes would align with the parallel `equipmentStock` array that loads each equipment into the game and makes their information accessible to display.

```
equipment              equipmentInventory
| 0 | telegramPtr |    | 0 | 3 |
| 1 | computerPtr |    | 1 | 0 |
```
Player has 3 telegrams, and no computers.


```suggestion
```